pkg/controller/configuration/configuration_test.go

```diff
@@ -82,35 +76,34 @@ func TestReconcile(t *testing.T) {
 		// If we attempt to create a Revision with a bad ConcurrencyModel set, we fail.
 		WantErr: true,
 		Objects: []runtime.Object{
-			setConcurrencyModel(cfg("validation-failure", "foo", 1234), "Bogus"),
+			builder.Config("validation-failure", "foo", 1234).WithConcurrencyModel("Bogus"),
 		},
 		WantCreates: []metav1.Object{
-			setRevConcurrencyModel(resources.MakeRevision(cfg("validation-failure", "foo", 1234), noBuildName), "Bogus"),
+			setRevConcurrencyModel(resources.MakeRevision(cfg("validation-failure", "foo", 1234)), "Bogus"),
```

Comment:
I definitely like the builders better than this style (`setRevConcurrencyModel`).